search/search-client-elasticsearch/src/main/java/io/camunda/search/transformers/ElasticsearchTransformers.java

```diff
@@ -0,0 +1,94 @@
+import io.camunda.search.clients.core.SearchQueryResponse;
+import io.camunda.search.clients.query.SearchBoolQuery;
+import io.camunda.search.clients.query.SearchConstantScoreQuery;
+import io.camunda.search.clients.query.SearchExistsQuery;
+import io.camunda.search.clients.query.SearchHasChildQuery;
+import io.camunda.search.clients.query.SearchIdsQuery;
+import io.camunda.search.clients.query.SearchMatchAllQuery;
+import io.camunda.search.clients.query.SearchMatchNoneQuery;
+import io.camunda.search.clients.query.SearchMatchQuery;
+import io.camunda.search.clients.query.SearchPrefixQuery;
+import io.camunda.search.clients.query.SearchQuery;
+import io.camunda.search.clients.query.SearchRangeQuery;
+import io.camunda.search.clients.query.SearchTermQuery;
+import io.camunda.search.clients.query.SearchTermsQuery;
+import io.camunda.search.clients.sort.SearchFieldSort;
+import io.camunda.search.clients.sort.SearchSortOptions;
+import io.camunda.search.clients.types.TypedValue;
+import io.camunda.search.transformers.query.BoolQueryTransformer;
+import io.camunda.search.transformers.query.ConstantScoreQueryTransformer;
+import io.camunda.search.transformers.query.ExistsQueryTransformer;
+import io.camunda.search.transformers.query.HasChildQueryTransformer;
+import io.camunda.search.transformers.query.IdsQueryTransformer;
+import io.camunda.search.transformers.query.MatchAllQueryTransformer;
+import io.camunda.search.transformers.query.MatchNoneQueryTransformer;
+import io.camunda.search.transformers.query.MatchQueryTransformer;
+import io.camunda.search.transformers.query.PrefixQueryTransformer;
+import io.camunda.search.transformers.query.QueryTransformer;
+import io.camunda.search.transformers.query.RangeQueryTransformer;
+import io.camunda.search.transformers.query.TermQueryTransformer;
+import io.camunda.search.transformers.query.TermsQueryTransformer;
+import io.camunda.search.transformers.search.SearchRequestTransformer;
+import io.camunda.search.transformers.sort.FieldSortTransformer;
+import io.camunda.search.transformers.sort.SortOptionsTransformer;
+import io.camunda.search.transformers.types.TypedValueTransformer;
+import java.util.HashMap;
+import java.util.Map;
+
+public final class ElasticsearchTransformers {
+
+  private final Map<Class<?>, SearchTransfomer<?, ?>> transformers;
+
+  public ElasticsearchTransformers() {
+    transformers = new HashMap<>();
+    initializeTransformers(this);
+  }
+
+  public <T, R> SearchTransfomer<T, R> getTransformer(final Class<?> cls) {
+    return (SearchTransfomer<T, R>) transformers.get(cls);
+  }
+
+  private void put(final Class<?> cls, final SearchTransfomer<?, ?> mapper) {
+    transformers.put(cls, mapper);
+  }
+
+  public static void initializeTransformers(final ElasticsearchTransformers mappers) {
+    // requests/response
+    mappers.put(SearchQueryRequest.class, new SearchRequestTransformer(mappers));
+    mappers.put(SearchQueryResponse.class, new SearchRequestTransformer(mappers));
```

Comment:
Is there any reason to use the request transformer on the response class?

Reply:
Again, copy paste error. It should of course use the response transformer.